modules/team_planner/spec/support/pages/team_planner.rb

```diff
@@ -343,12 +343,7 @@ def y_center(element)
 
     def wait_for_loaded
       expect(page).to have_css('.op-team-planner--wp-loading-skeleton')
-
-      retry_block do
-        raise "Should not be there" if page.has_selector?('.op-team-planner--wp-loading-skeleton')
-
-        sleep 5
-      end
+      expect(page).to have_no_css('.op-team-planner--wp-loading-skeleton')
```

Comment:
Perhaps extend the wait on this? I've seen this retry twice/thrice quite consistently on the CI logs with the previous implementation so it takes at least 5-10 seconds for it to be gone. Maybe:

```suggestion
      expect(page).to have_no_css('.op-team-planner--wp-loading-skeleton', wait: 10)
```